recipes/basemap-data-hires/bld.bat

```diff
@@ -0,0 +1,2 @@
+copy %SRC_DIR%\lib\mpl_toolkits\basemap\data\*_h.dat %LIBRARY_PREFIX%\share\basemap || exit 1
+copy %SRC_DIR%\lib\mpl_toolkits\basemap\data\*_f.dat %LIBRARY_PREFIX%\share\basemap || exit 1
```

Comment:
If ~30MB is still big we can put the intermediary data resolution here too. Right now it is just the 'full' and the 'high' resolutions.